.gitignore

```diff
@@ -45,3 +45,5 @@
 /packages
 
 launchSettings.json
+
+/Dist/copyBIOS.sh

```

Comment:
I presume this and `{firmware,lua}.zip` are things you were using for debugging?

Reply:
Yes I was using copyBIOS.sh for testing but it wasn't useful anymore so I removed it. I will also remove it from gitignore. for ``{firmware,lua}.zip`` it's not something I added though so I'm not sure